layouts/partials/block/data.html

```diff
@@ -52,6 +52,7 @@
     */}}
     {{ $newSrc = replace $newSrc "/tree/main" "/readme" }}
     {{ $newSrc = replace $newSrc "/blob/main/README.md" "/readme" }}
+    {{ $newSrc = "https://api.github.com/repos/CodeYourFuture/CYF-Workshops/readme/git-day-1?ref=rewrite/git-day-1" }}
```

Comment:
```suggestion
    {{ $newSrc = "https://api.github.com/repos/CodeYourFuture/CYF-Workshops/readme/git-day-1?ref=rewrite/git-day-1" }}
```

Looks like you've hardcoded in a single source, which doesn't seem desirable. ;P 
```suggestion

```